### PR TITLE
cast search_objects and mapping to ElasticResult

### PR DESCRIPTION
**Pull request by benghaziboy**
@mparent61

### CODE REVIEW

File: djes/factory.py [hunk 1]
```diff
@@ -73,9 +73,16 @@ class Meta(object):
             proxy = True
             app_label = model._meta.app_label
 
+        class ElasticMapping:
+            class Meta:
+                elastic = True
```
Comment (mparent61) [inline djes/factory.py]: I'd make this marker var more explicit, like `is_base_elastic_mapping`

**Comment by mparent61** [comment]
LGTM

File: djes/factory.py (hunk 1 above)
Comment (benghaziboy) [inline djes/factory.py]: how about `elastic_abstract` since we don't perform operation on these classes.

Comment (mparent61) [inline djes/factory.py]: yes

**Pull request merged**